### ISSUE TITLE
The requested module '../lib/application.js' does not provide an export named 'default

### ISSUE THREAD

**Issue by aladdin-add** [esbuild-kit/esno#25]
I was trying to use it in a koa app, but got the error:

```sh
> repros@1.0.0 dev
> esno serve.ts

(node:23537) ExperimentalWarning: Custom ESM Loaders is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
SyntaxError: The requested module '../lib/application.js' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:128:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:194:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:409:24)
    at async loadESM (node:internal/process/esm_loader:85:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)
```

seem esno does not support import cjs in a esm(it has been supported in node.js)?
https://unpkg.com/browse/koa@2.13.4/dist/koa.mjs

env:

esno: 0.15.0
node: v18.1.0

**Comment by aladdin-add** [comment]
repro: https://github.com/aladdin-add/repros/tree/esno-issue25

**Comment by privatenumber** [comment]
Thanks for the report and reproduction.

Should be fixed in [`tsx@^3.0.0`](https://github.com/esbuild-kit/tsx/releases/tag/v3.1.0)!
